src/activities/categorization/Categorization.qml

```diff
@@ -62,26 +62,29 @@ ActivityBase {
             property alias menuScreen: menuScreen
             property alias menuModel: menuScreen.menuModel
             property alias dialogActivityConfig: dialogActivityConfig
-            property string mode: "easy"
+            property bool scoreVisible: true
             property bool instructionsVisible: true
-            property bool categoryImageChecked: (mode === "easy" || mode === "medium")
-            property bool scoreChecked: (mode === "easy")
-            property bool iAmReadyChecked: (mode === "expert")
-            property bool displayUpdateDialogAtStart: true
+            property bool categoryImageVisible: true
+            property bool categoryImageChecked: categoryImageVisible
+            property bool scoreChecked: scoreVisible
+            property bool instructionsChecked: instructionsVisible
+            property bool iAmReadyChecked: ((!instructionsVisible && !scoreVisible) || (!instructionsVisible && scoreVisible))
```

Comment:
!instructionsVisible only?
Why don't you use it for iAmReady visible which is the same if I'm not wrong?